javascript/javascript1/week4/week4-homework/getcommand.js

```diff
@@ -0,0 +1,133 @@
+"use strict"
+
+
+
+let names = [];
+let whoAmI = 'What is my name';
+let day = 'What day is it today'
+let todo =[];
+let myTodoList = 'What is on my todo'
+let favDish = [];
+let event = [];
+let eventList = 'What am I doing this week'
+
+
+function getReply(command) {
```

Comment:
What would happen if I call this function two times like this ?
`console.log(getReply('Hello my name is Benjamin'))
console.log(getReply('Hello my name is Benjamin'))
`
Can you spot the problem ?

Reply:
It'll log out 'nice to meet you Benjamin' 2x
I see the problem, I'll fix it.